packages/zowe-explorer-api/src/vscode/ZoweVsCodeExtension.ts

```diff
@@ -46,21 +47,25 @@ export class ZoweVsCodeExtension {
     }
 
     /**
-     * Reveal an error in VSCode, and log the error to the Imperative log
+     * Show a message within VS Code dialog, and log it to an Imperative logger
+     * @param message The message to display
+     * @param severity The level of severity for the message (see `MessageSeverity`)
+     * @param logger The IZoweLogger object for logging the message
      *
+     * @deprecated Please use `Gui.showMessage` instead
      */
-    public static showVsCodeMessage(message: string, severity: MessageSeverityEnum, logger: IZoweLogger): void {
-        logger.logImperativeMessage(message, severity);
-
-        const errorMessage = `${logger.getExtensionName()}: ${message}`;
+    public static showVsCodeMessage(message: string, severity: MessageSeverity, logger: IZoweLogger): void {
+        void Gui.showMessage(message, { severity: severity, logger: logger });
```

Comment:
I'm still trying to understand the difference between using the `void` vs `await`.
I believe both may perform a void statement in practice.

I guess I don't see a problem with statements like these
```javascript
undefined; // same as `await promise-void`
1234; // same as `await promise-number`
true; // same as `await promise-boolean`
```

Reply:
This is essentially mimicking the original behavior of `showVsCodeMessage` (screenshot above), but we can definitely update the function to not use `void`.